footer.php

```diff
@@ -1,26 +1,26 @@
 			<div id="footer">
 				<div class="row">
 					<div class="span12 text-center">
-						<?=wp_nav_menu( array(
-							'theme_location' => 'social-links', 
+						<?php echo wp_nav_menu( array (
```

Comment:
`echo` isn't necessary when calling `wp_nav_menu()`; this function echo's its results by default